tests/feature_extraction/test_text.py

```diff
@@ -107,3 +107,17 @@ def test_correct_meta():
     assert scipy.sparse.issparse(result._meta)
     assert result._meta.dtype == "float64"
     assert result._meta.shape == (0, 0)
+
+
+def test_count_vectorizer():
+    # TODO: gen_cluster, pickle futures, issue.
+    from distributed import Client
+
+    with Client():
```

Comment:
Either gen_cluster (recommended) or the `c, s, a, b` fixtures in utils_test which support the sync API